test-utils/testlib/src/fees_utils.rs

```diff
@@ -155,11 +163,27 @@ impl FeeHelper {
         self.gas_to_balance(exec_gas + send_gas)
     }
 
-    pub fn delete_account_cost(&self) -> Balance {
+    pub fn prepaid_delete_account_cost(
+        &self,
+        #[cfg(feature = "protocol_feature_allow_create_account_on_delete")]
+        implicit_account_created: bool,
```

Comment:
Code smell: this boolean is always passed as literal `true` or `false` on the call site. This means that it might be better to split this function in two with different names:

```rust
pub fn prepaid_delete_account_cost_for_implicit_account(&self) {
    self.prepaid_delete_account_cost(false)
}
pub fn prepaid_delete_account_cost_for_explicit_account(&self) {
    self.prepaid_delete_account_cost(true)
}
fn self.prepaid_delete_account_cost(&self, is_accout_implicet) {
    ...
}

```

Reply:
Done